final_results/result_processing_wrapper.sh

```diff
@@ -36,21 +36,52 @@ else
   REGEX="java-1.8.0"
 fi
 
+graph_parameters() {
+##GRAPH_NAME="LOCAL" ##necessary for graph naming
+echo "running graph parameters"
+if [[ ($1 == *"local"*) ]];then
+  graph_name=local"$2"
+else
+  graph_name=virtual"$2"
```

Comment:
This will be fragile for nested vms. I would recomendto chaneg it like: 
```
if [[ ($1 == *"local"*) ]];then
  graph_name=local"$2"
elif [[ ($1 == *"local_vm"*) ]];then #not sure what exactly
  graph_name=singlevm_vml"$2" #not sure what exactly
else
 echo "unnown locality $2"
 exit 1
fi